library/ssl_tls.c

```diff
@@ -948,7 +948,7 @@ static int ssl_populate_transform( mbedtls_ssl_transform *transform,
 
 #if defined(MBEDTLS_SSL_SOME_SUITES_USE_MAC)
 #if defined(MBEDTLS_SSL_PROTO_TLS1_2)
-    if( minor_ver >= MBEDTLS_SSL_MINOR_VERSION_1 )
+    if( minor_ver == MBEDTLS_SSL_MINOR_VERSION_3 )
```

Comment:
`>=` instead of `==`

Reply:
Same as above (and all `>=` vs `==` comments): either the check is useless, or the preprocessor directive above is wrong.